### PR TITLE
Improve Cache Busting for CSS and Javascript files

### LINKED ISSUE #201: Improve Cache Busting for CSS and Javascript files

### PR DESCRIPTION
**Pull request by sbrack8t**
Fixes #201

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -69,33 +69,33 @@ function gesso_scripts() {
...
+  wp_register_script('gessocommon', get_template_directory_uri() . '/js/dist/common.min.js', array('jquery'), filemtime(get_template_directory_uri() . '/js/dist/common.min.js' ) );
   wp_enqueue_script('gessocommon');
 
-  wp_register_script('gessomodernizr', get_template_directory_uri() . '/js/libraries/modernizr.min.js', array('jquery'), $gesso_version ); // Modernizr
+  wp_register_script('gessomodernizr', get_template_directory_uri() . '/js/libraries/modernizr.min.js', array('jquery'), filemtime(get_template_directory_uri() .'/js/libraries/modernizr.min.js' ) ); // Modernizr
   wp_enqueue_script('gessomodernizr');
 
-  wp_register_script('gessodetailspolyfill', get_template_directory_uri() . '/js/libraries/details-element-polyfill.js', array('jquery'), $gesso_version ); // Details polyfill for IE/Edge
+  wp_register_script('gessodetailspolyfill', get_template_directory_uri() . '/js/libraries/details-element-polyfill.js', array('jquery'), filemtime(get_template_directory_uri() . '/js/libraries/details-element-polyfill.js' ) ); // Details polyfill for IE/Edge
```
Comment (timnolte) [inline functions.php]: @sbrack8t FYI, when using the `filemtime` you need to use `get_template_directory()` to get the filesystem path and not  the URL.

File: functions.php [hunk 2]
```diff
@@ -69,33 +69,33 @@ function gesso_scripts() {
...
-  wp_register_script('gessoscripts', get_template_directory_uri() . '/js/dist/scripts.min.js', array('jquery','gessocommon','gessomodernizr'), $gesso_version ); // Custom scripts
+  wp_register_script('gessoscripts', get_template_directory_uri() . '/js/dist/scripts.min.js', array('jquery','gessocommon','gessomodernizr'), filemtime(get_template_directory_uri() . '/js/dist/scripts.min.js') ); // Custom scripts
   wp_enqueue_script('gessoscripts');
 
   // Add Google Fonts
   wp_register_style('gessoFonts', 'https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i&display=swap', $gesso_version );
   wp_enqueue_style( 'gessoFonts');
 
-  wp_enqueue_style( 'style', get_stylesheet_directory_uri() . '/css/styles.css', array(), $gesso_version, 'all' );
+  wp_enqueue_style( 'style', get_stylesheet_directory_uri() . '/css/styles.css', array(), filemtime( get_template_directory_uri() . '/css/styles.css' ), 'all' );
```
Comment (timnolte) [inline functions.php]: @sbrack8t this should be using `get_sylesheet_directory()` when referencing stylesheets.

**Review by timnolte — CHANGES_REQUESTED** [review]
@sbrack8t this should be go once those `filemtime` calls are fixed per my inline comments.

File: functions.php (hunk 1 above)
Comment (sbrack8t) [reply functions.php]: Updated

File: functions.php (hunk 2 above)
Comment (sbrack8t) [reply functions.php]: Updated

**Review by timnolte — APPROVED** [review]
Looks good.

**Pull request merged**